src/meta/meta_ckpt.c

```diff
@@ -427,23 +439,44 @@ __wt_meta_ckptlist_set(WT_SESSION_IMPL *session,
 			__wt_seconds(session, &secs);
 			ckpt->sec = (uintmax_t)secs;
 		}
+
+		__wt_timestamp_addr_check(session, ckpt->oldest_start_ts,
+		    ckpt->newest_start_ts, ckpt->newest_stop_ts);
 		if (strcmp(ckpt->name, WT_CHECKPOINT) == 0)
 			WT_ERR(__wt_buf_catfmt(session, buf,
 			    "%s%s.%" PRId64 "=(addr=\"%.*s\",order=%" PRId64
-			    ",time=%" PRIuMAX ",size=%" PRIu64
+			    ",time=%" PRIuMAX
+			    ",size=%" PRIu64
+			    ",oldest_start=%" PRIu64
+			    ",newest_start=%" PRIu64
+			    ",newest_stop=%" PRIu64
```

Comment:
I think these should all have a `_ts` suffix to match the strings you're looking for in `__ckpt_load`.

Reply:
Thank you, @sueloverso!

And the fact I didn't catch this means I forgot to hook the checkpoint entries into the verify code.